samples/core/Saving/Saving/Transactions/AmbientTransaction/Sample.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.EntityFrameworkCore;
+using System.Data.SqlClient;
+using System.Transactions;
+
+namespace EFSaving.Transactions.AmbientTransaction
+{
+    public class Sample
+    {
+        public static void Run()
+        {
+            var connectionString = @"Server=(localdb)\mssqllocaldb;Database=EFSaving.Transactions;Trusted_Connection=True;ConnectRetryCount=0";
+
+            using (var context = new BloggingContext(
+                new DbContextOptionsBuilder<BloggingContext>()
+                    .UseSqlServer(connectionString)
+                    .Options))
+            {
+                context.Database.EnsureDeleted();
+                context.Database.EnsureCreated();
+            }
+
+            #region Transaction
+            using (var scope = new TransactionScope(TransactionScopeOption.Required))
```

Comment:
Mind if I change the isolation level? The default isn't great. See @davidbaxterbrowne's write up for reference: https://blogs.msdn.microsoft.com/dbrowne/2010/06/03/using-new-transactionscope-considered-harmful/